### ISSUE TITLE
Linebreaks in map Symbol Layer seems to have been removed

### ISSUE THREAD

**Issue by gusbo** [visgl/react-map-gl#1558]
#### Description

Linebreaks in map Symbol Layer seems to have been removed. This means multi-line labels are not displayed correctly.

#### Repro Steps

Create a map example with a Style containing a symbol layer for a text label containing a format expression containing linebreaks like so:
`{
      "id": "labels_long_right",
      "type": "symbol",
      "source": "tiles",
      "source-layer": "line_view",
      "minzoom": 6,
      "filter": [
        "all",
        [
          "==",
          "symbol_name",
          "labelf2"
        ]
      ],
      "layout": {
        "text-field": [
          "format",
          "88",
          {
            "text-color": "black"
          },
          "\n",
          [
            "get",
            "name"
          ],
          {
            "text-color": "white"
          },
          "\n",
          "FL097",
          {
            "text-color": "red"
          }
        ],
        "symbol-placement": "line-center",
        "icon-text-fit": "none",
        "visibility": "visible",
        "icon-padding": 2,
        "icon-offset": [
          0,
          0
        ],
        "icon-text-fit-padding": [
          1,
          1,
          1,
          2
        ],
        "text-offset": [
          -0.5,
          0
        ],
        "text-max-angle": 0,
        "icon-allow-overlap": true,
        "icon-ignore-placement": true,
        "icon-image": "iconlabelf2",
        "icon-size": 0.8
      },`

This will result in labels being shown with correct colors, but without the linebreaks.
The expected behaviour is that the label should contain several lines on top of each other:
88
T402
FL097

The exact same style works correctly in Maputnik and on Android and iOS using React Native.

#### Environment (please complete the following information):

- Library Version: react-map-gl: "6.1.16"
- Mapbox Version: mapbox-gl:version "2.4.1"
- React Version: 17.0.2
- Browser Version: Chrome 92.0.4515.159
- OS: Mac OS X 11.5.2 

#### Logs

Unsure if this error message from the Chrome Console is relevant, but including it anyway:

Unable to perform style diff: Unimplemented: setSprite, setGlyphs..  Rebuilding the style from scratch.
B @ util.js:516
_updateDiff @ map.js:1570
(anonymous) @ map.js:1556
(anonymous) @ ajax.js:180
Promise.then (async)
c2 @ ajax.js:169
(anonymous) @ ajax.js:150
Promise.then (async)
u2 @ ajax.js:147
(anonymous) @ ajax.js:189
At @ ajax.js:194
t.getJSON @ ajax.js:256
_diffStyle @ map.js:1552
setStyle @ map.js:1501
_updateMapStyle @ mapbox.js:358
_update @ mapbox.js:346
setProps @ mapbox.js:170
(anonymous) @ static-map.js:180
commitHookEffectListMount @ react-dom.development.js:20573
commitLifeCycles @ react-dom.development.js:20634
commitLayoutEffects @ react-dom.development.js:23426
callCallback2 @ react-dom.development.js:3945
invokeGuardedCallbackDev @ react-dom.development.js:3994
invokeGuardedCallback @ react-dom.development.js:4056
commitRootImpl @ react-dom.development.js:23151
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
commitRoot @ react-dom.development.js:22990
performSyncWorkOnRoot @ react-dom.development.js:22329
(anonymous) @ react-dom.development.js:11327
unstable_runWithPriority @ scheduler.development.js:468
runWithPriority$1 @ react-dom.development.js:11276
flushSyncCallbackQueueImpl @ react-dom.development.js:11322
flushSyncCallbackQueue @ react-dom.development.js:11309
discreteUpdates$1 @ react-dom.development.js:22420
discreteUpdates @ react-dom.development.js:3756
dispatchDiscreteEvent @ react-dom.development.js:5889

**Comment by Pessimistress** [comment]
Styling is handled by mapbox-gl, not this library. You should report your issue in the Mapbox GL JS repo.

**Comment by gusbo** [comment]
For information: I did some more testing to try to reproduce the behaviour using only mapbox-gl and standard React. The problem can be fixed by using setRTLPlugin(). I guess this should be possible using react-map-gl as well?

**Comment by Pessimistress** [comment]
https://visgl.github.io/react-map-gl/search
